### ISSUE TITLE
When you gonna fix this issue? 

### ISSUE THREAD

**Issue by ykcodes** [micheleg/dash-to-dock#1293]
Why there are two docks.

<img src="https://i.imgur.com/oIka3Xe.png">

**Comment by lepa22** [comment]
Restart GNOME Shell using <kbd>Alt</kbd>+<kbd>F2</kbd>, input `r` and press <kbd>Enter</kbd>.

**Comment by paroj** [comment]
> Why there are two docks.

because you did not disable the ubuntu one. its your own fault. :clinking_glasses:


<details>

```sh
gnome-extensions disable ubuntu-dock@ubuntu.com
```

</details>

**Comment by ykcodes** [comment]
It's already disabled, 

<img src="https://i.imgur.com/I15AeHr.png">

**Comment by paroj** [comment]
maybe this only works on Ubuntu 20.04. Anyway this is a duplicate of #649

**Comment by NESC1US** [comment]
yes, i had that as well in 20.04, do this part as well and it will go away
gnome-extensions disable ubuntu-dock@ubuntu.com
sudo apt remove gnome-shell-extension-ubuntu-dock

**Comment by DeadlyDinosaurs** [comment]
It's not working for me, i tried in both 18.04 and 20.04 after I'm restarting the computer this things is still getting the same issue.

 
![image](https://user-images.githubusercontent.com/61355855/96967199-c2731b80-152c-11eb-9ea5-6365f40c8116.png)

so every time <kbd>Alt</kbd> + <kbd>F2</kbd> and then restart the shell by typing **r** or **restart**

**Comment by DJMucki** [comment]
> yes, i had that as well in 20.04, do this part as well and it will go away
> 
> ```
> gnome-extensions disable ubuntu-dock@ubuntu.com
> sudo apt remove gnome-shell-extension-ubuntu-dock
> ```

This worked for me. 
Thanks
